src/Generator/ValueGenerator.php

```diff
@@ -143,7 +149,7 @@ public function deleteConstant($constant)
     /**
      * Return constant list
      *
-     * @return ArrayObject
+     * @return SplArrayObject|StdlibArrayObject
```

Comment:
@Maks3w — what was that in reply to? using the SPL ArrayObject only? having `Zend\Stdlib\ArrayObject` extend `ArrayObject`? or removing the `ArrayAccess` typehint in the constructor?